pkg/cliutil/uri.go

```diff
@@ -91,16 +91,26 @@ func GetIngressHost(ctx context.Context, proxyName, proxyNamespace, proxyPort st
 	}
 
 	var host, port string
-	// gateway-proxy is an externally load-balanced service
-	if len(svc.Status.LoadBalancer.Ingress) == 0 || localCluster {
-		// assume nodeport on kubernetes
+	serviceType := svc.Spec.Type
+	if localCluster {
+		serviceType = v1.ServiceTypeNodePort
+	}
+	switch serviceType {
+	case v1.ServiceTypeClusterIP:
+		fmt.Println("Warning: Potentially invalid proxy configuration service type of ClusterIP")
```

Comment:
Perhaps this is unnecessary since we have the PR threads as context, but I wonder if a comment in the code here would be valuable outlining why this is invalid configuration, and why we are still setting the host/port and not erroring. @nfuden thoughts?

Reply:
Im always a fan of more comments but in this case since its a cli warning I tend to think we should make sure that the log line we are emitting has at least 90% of the context as otherwise we will have users who will wonder the same question.

In that case I propose  the following if it works for you Sam
```suggestion
		// There are a few edge cases where glooctl could be run in an environment where this is not a fatal error
		// However the service type ClusterIP does not accept incoming traffic which doesnt work as a ingress
		logger := GetLogger()
		logger.Write([]byte(""Warning: Potentially invalid proxy configuration, proxy may not accepting incoming connections"))
```

I am thinking this because then we can pull out that warning in the future to be a const so that we eventually support multi language and reuse of the invalid proxy configuration line.